code/online_challenges/src/codechef/COINS/coins.cpp

```diff
@@ -0,0 +1,34 @@
+#include<bits/stdc++.h>
```

Comment:
#1915. Remove using statement. Reformat this file using clang-format with 4 spaces.

Reply:
OK..

Reply:
Sir I have done changes please review them.